services/dy-3dvis/simcoreparaviewweb/config/apache.conf

```diff
@@ -0,0 +1,24 @@
+  SetEnv PARAVIEW_INPUT_PATH ${PARAVIEW_INPUT_PATH}
+  SetEnv POSTGRES_DB ${POSTGRES_DB}
+  SetEnv POSTGRES_USER ${POSTGRES_USER}
+  SetEnv POSTGRES_PASSWORD ${POSTGRES_PASSWORD}
+  SetEnv POSTGRES_ENDPOINT ${POSTGRES_ENDPOINT}
+  SetEnv S3_BUCKET_NAME ${S3_BUCKET_NAME}
+  SetEnv S3_SECRET_KEY ${S3_SECRET_KEY}
+  SetEnv S3_ACCESS_KEY ${S3_ACCESS_KEY}
+  SetEnv S3_ENDPOINT ${S3_ENDPOINT}
+  SetEnv SIMCORE_NODE_UUID ${SIMCORE_NODE_UUID}
+  SetEnv PYTHONPATH ${PYTHONPATH}
+  
+  Alias "/setport" "/home/root/src/setport.py"
+  Alias "/retrieve" "/home/root/src/input-retriever.py"
```

Comment:
This looks cool. Is it secure? Not that I really care, just out of interest...

Reply:
secure? sure;) I guess as much as apache2 is...